test/unit/utMatrix3x3.cpp

```diff
@@ -73,9 +73,21 @@ TEST_F(utMatrix3x3Test, FromToMatrixTest) {
 
     aiVector3D from, to;
 
+    auto random_ratio = []() -> float {
+        return static_cast<float>(rand() / RAND_MAX);
+    };
+
     for (int i = 0; i < NUM_SAMPLES; ++i) {
-        from = aiVector3D(1.f * rand() / RAND_MAX, 1.f * rand() / RAND_MAX, 1.f * rand() / RAND_MAX).Normalize();
-        to = aiVector3D(1.f * rand() / RAND_MAX, 1.f * rand() / RAND_MAX, 1.f * rand() / RAND_MAX).Normalize();
```

Comment:
This will trigger implicit int -> float conversion error around large integer values

```
/home/hbina/git/assimp/test/unit/utMatrix3x3.cpp:77:42: warning: implicit conversion from 'int' to 'float' changes value from 2147483647 to 2147483648 [-Wimplicit-int-float-conversion]
[build]         from = aiVector3D(1.f * rand() / RAND_MAX, 1.f * rand() / RAND_MAX, 1.f * rand() / RAND_MAX).Normalize();
[build]                                        ~ ^~~~~~~~
```

Can be silenced by explicitly casting them.